src/Dumper.cpp

```diff
@@ -266,7 +265,7 @@ void Dumper::dumpString(char const* src, ValueLength len) {
   static char const EscapeTable[256] = {
       // 0    1    2    3    4    5    6    7    8    9    A    B    C    D    E
       // F
-      'u',  'u', 'u', 'u', 'u', 'u', 'u', 'u', 'b', 't', 'n', 'u', 'f', 'r',
+      'u',  'u', 'u', 'u', 'u', 'u', 'u', 'u', 'u', 't', 'n', 'u', 'u', 'r',
```

Comment:
What is the motivation for this change?
This will make `\b` and `\f` be represented as `\u0008` and `\u000C` instead of `\b` and `\f`.
But `\b` and `\f` are perfectly valid, and shorter than their `\u...` equivalents.
https://www.json.org/json-en.html